content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -102,44 +167,134 @@ Adding tags to tabular metrics is similar to adding global tags, with two additi
+2. Click **Create New Tag** at the top of the modal.
+3. Select the dropdown in the search field to add the OID name.
+4. Click the **Modification** dropdown to add a modification. See advanced [options](?tab=manual#global-tags).
+5. Click **Create** to save the new tag.
+6. This returns you to the define global tags screen where you can see the new tag that was added.
+
+   {{< img src="/network_device_monitoring/profile_onboarding/add_global_tags_manual.mp4" alt="Video showing the add global tags modal, adding a new tag with the manual method, and returning to global tags step." video=true >}}
+
+{{% collapse-content title="Advanced options" level="h4" expanded=false %}}
+
+#### Global tags:
+
+| Modification    | Description                                                                                         |
+|-----------------|-----------------------------------------------------------------------------------------------------|
+| No Modification | The device's returned value will be used directly as the tag value.                                 |
+| Format          | This can be [mac_address][5] or [ip_address][6].                                                    |
+| Extract Value   | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |
+| Mapping         | This is described [here][8].   
+
+[5]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#format-mac_address
+[6]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#format-ip_address
+[7]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#extract_value
+[8]: https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#mapping-index-to-tag-string-value
+
+{{% /collapse-content %}}
+
+{{% /tab %}}
+{{< /tabs >}}
 
 ### Apply a profile to created devices
 
-1. After you save a profile, navigate back to the [profile home page][4] and select the **Download All Profiles** option. This allows you to download the `.zip` bundle which contains the `yaml` files for the profiles you created. 
+{{< tabs >}}
+{{% tab "Automatic (recommended)" %}}
+
+After applying your configuration options to your device profile, click **Save and Sync Agents** to automatically apply this profile to all NDM agents. The configurations are applied to your devices with [Remote Configuration][14].
+
+{{< img src="/network_device_monitoring/profile_onboarding/save_sync_agents.png" alt="The Network Device profile page showing the final step to save and sync agents" style="width:100%;">}}
+
+[14]: /agent/remote_config
+
+{{% /tab %}}
+
+{{% tab "Manual" %}}
+
+1. After you save a profile as a draft, navigate back to the [profile home page][4], select the **Draft Profiles** tab, then select the **Download All Profiles** option. This allows you to download the `.zip` bundle which contains the `yaml` files for the profiles you created. 
```

Comment:
The draft mode is used as a way to facilitate the profile creation process for customers since it can be a long flow, they can use the draft mode when the profile is still in progress (ie not ready to be synced and applied to agents).
Draft profiles are excluded from any real operation like the sync to agents or the download.
So any profile marked as draft **won't be** downloaded.
Also that the download option means that **all** profiles will be downloaded in a zip together. Customers can't chose what profiles to download no matter what tab they select.